astropy/modeling/optimizers.py

```diff
@@ -216,11 +216,10 @@ def __call__(self, objfunc, initval, fargs, **kwargs):
         """
         if 'maxiter' not in kwargs:
             kwargs['maxiter'] = self._maxiter
-        if 'acc' not in kwargs:
-            kwargs['acc'] = self._acc
+  
```

Comment:
Oh, I see now that `self._acc` is passed as `xtol`. 
I'm not sure I understand what you mean exactly. `acc` is present in fmin_slsqp, while `xtol` and `ftol` are in fmin (=Simplex). Now the choice is whether to go for `acc` in all calls, and transform it to the relevant parameter of the minimizer inside these functions, or to expose the scipy keywords, do I understand correctly?